impl/src/main/java/com/sun/faces/application/resource/WebappResourceHelper.java

```diff
@@ -65,7 +67,7 @@ public WebappResourceHelper() {
 
         WebConfiguration webconfig = WebConfiguration.getInstance();
         cacheTimestamp = webconfig.isOptionEnabled(CacheResourceModificationTimestamp);
-        BASE_RESOURCE_PATH = webconfig.getOptionValue(WebConfiguration.WebContextInitParameter.WebAppResourcesDirectory);
+        BASE_RESOURCE_PATH = ensureLeadingSlash(webconfig.getOptionValue(WebConfiguration.WebContextInitParameter.WebAppResourcesDirectory));
         BASE_CONTRACTS_PATH = webconfig.getOptionValue(WebConfiguration.WebContextInitParameter.WebAppContractsDirectory);
```

Comment:
It looks like BASE_CONTRACTS_PATH also needs this fix.

Reply:
Yes I already wanted to take a closer look at it later.